client/components/???Cart.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
```

Comment:
We should never have code in the main branch that is questionable or un-used. If there's any question as to whether something is right, it should stay in a branch until there's no question that it's right. There's a handful of other files in the repo that look like this. Those should all be removed, and if something doesn't work it should get its own ticket